src/skybox.rs

```diff
@@ -15,11 +15,11 @@ use bevy::{
 pub struct AtmosphereSkyBoxMaterial(pub Handle<SkyBoxMaterial>);
 
 /// The `Handle` for the shader for the [`SkyBoxMaterial`].
-pub const ATMOSPHERE_SKYBOX_SHADER_HANDLE: HandleUntyped =
-    HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 4511926918914205353);
+pub const ATMOSPHERE_SKYBOX_SHADER_HANDLE: Handle<Shader> =
```

Comment:
Could you use newly added `embedded_asset!` instead?

Reply:
I just gave that a go, but am running into a panic from an unwrap with `embedded_asset!`, when used like this `        embedded_asset!(app, "shaders/skybox.wgsl");` in `plugin.rs`. I will take a deeper look tomorrow and hopefully push up a fix

Reply:
And just as a posted, I figured it out. Bevy does some slightly strange path manipulation using `file!()`, and it assumes that `file!()` will always return with a leading `/src/`, but that doesn't seem to be the case all of the time. Luckily you can work around this issue with: `embedded_asset!(app, "src/", "shaders/skybox.wgsl");`

Reply:
Great!